utilities/functions.py

```diff
@@ -1,7 +1,40 @@
 import cv2
 import numpy as np
+import math
 
 
+def scale_value(
+    value: float,
+    input_lower: float,
+    input_upper: float,
+    output_lower: float,
+    output_upper: float,
+    exponent: float = 1,
```

Comment:
Why does this function also raise to a power? That is a slow operation, that won't be needed most of the time, and it's easy enough to apply to the result. I would remove it.

Reply:
I've made a check for whether or not it should do the exponent to hopefully save some time. The reason for this is that in past we have wanted to have an exponent built in to the function.